src/pages/Feeds/Feed.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+
+function getTarget(elem, className) {
+  while (!elem.classList.contains(className)) {
+    elem = elem.parentNode;
+
+    if (elem.nodeName === 'BODY') {
+      elem = null;
+      return;
+    }
+  }
+  return elem;
+}
+
+class Feed extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      curTranslateX: 0,
+      isOverflow: false,
+    };
+  }
+
+  carouselLeaveMouse = event => {
+    this.setState({
+      isOverflow: false,
+    });
+  };
+
+  carouselOverMouse = event => {
+    this.setState({
+      isOverflow: true,
+    });
+  };
+
+  moveToX = event => {
+    let { curTranslateX } = this.state;
+    let { name } = getTarget(event.target, 'carouselControl');
+
+    name === 'right'
+      ? this.setState({ curTranslateX: curTranslateX - 50 })
+      : this.setState({ curTranslateX: curTranslateX + 50 });
+  };
+
+  render() {
+    let { curTranslateX, isOverflow } = this.state;
+    let { feeds } = this.props;
+
+    return (
+      <>
+        <div
+          onMouseOver={this.carouselOverMouse}
+          onMouseLeave={this.carouselLeaveMouse}
+          className="feedsCarousel"
+          style={isOverflow ? { overflow: 'auto' } : { overflow: 'hidden' }}
```

Comment:
아래와 같이 조건문 위치에 따라서 코드 반복을 줄일 수 있습니다!
```suggestion
          style={{ overflow: isOverflow ? 'auto' : 'hidden' }}
```

Reply:
멘토님께서 금요일에 말씀해주신 바와 같이 처음부터 overflow를 scroll로 해줌으로써, 해당 코드 부분은 불필요하여 삭제하였습니다!